O/OptFrame/build_tarballs.jl

```diff
@@ -0,0 +1,43 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name    = "OptFrame"
+version = v"6.0.0"
```

Comment:
Why this version number? The latest release is 5.1.0, and you are building from a more recent commit on master. If you use this version number now, then you can't use it when 6.0 is actually released.